examples/ok-with-rename/README.md

```diff
@@ -0,0 +1,3 @@
+# One-off rename
+
+This example shows how to rename a single file in the doc site.
```

Comment:
````suggestion
This example shows how to rename a single file in the doc site.

## Configuration

mkdocs.yml

```yaml
{% include "examples/ok-with-rename/mkdocs-test.yml" %}
```

Folder structure:

```
project
│   README.md
|   foo.bar
```

## Output

```
project
│   README.md
|   foo.bar
|
└───site
│   │   index.html
|   |    
|   └───baz
|         │   index.html
```

````

Reply:
Looks fine to me; I am hitting the "Commit suggestion" button to see if it works the way I hope/presume it will...